opdracht-2/styles.css

```
@@ -0,0 +1,172 @@
+/*////////////////////////////////////////////////////////////////////////
+Table of Contents:
+    1. Global Styles
+    2. Typografie
+    3. Navigation
+    4. Web Page Content
+     Media Queries
+////////////////////////////////////////////////////////////////////////*/
+
+/*////////////////////////
+    1. Global Styles
+////////////////////////*/
+
+* {
+    box-sizing: border-box;
+}
+
+html {
+    height: 100%
+}
+
+body {
+    display: flex;
+    flex-direction: row;
+    height: 100%
+}
+
+/*////////////////////////
+    2. Typography
+////////////////////////*/
+
+.nav-bar {
+    font-size: 18px;
+    font-weight: bold;
+    color: #383838
+}
+
+/*////////////////////////
+     3. Navigation
+////////////////////////*/
+
+.nav-bar {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    background-color: #f3f3f3;
+    flex-basis: 250px;
+    flex-grow: 1;
```

```suggestion
    flex-grow: 1;
    flex-shrink: 0;
```
Voeg ook flex-shrink: 0; toe zodat de menu-balk nooit kleiner wordt dan 250 pixels. flex-shrink heeft een standaard waarde van 1 als je deze niet aanpast.